docs/_docs/plugins/installation.md

```diff
@@ -3,77 +3,123 @@ title: Plugins
+Jekyll has built-in support for using plugins to extend the core functionality.
 
-1. In your site source root, make a `_plugins` directory. Place your plugins
-   here. Any file ending in `*.rb` inside this directory will be loaded before
-   Jekyll generates your site.
+Primarily, any file with extension `.rb` placed within a `_plugins` directory at the root of the site's `source`, will be automatically loaded
+during a build session.
 
-2. In your `_config.yml` file, add a new array with the key `plugins` (or `gems` for Jekyll < `3.5.0`) and the
-   values of the gem names of the plugins you'd like to use. An example:
+This behavior can be configured as per following:
 
-   ```yaml
-   # This will require each of these plugins automatically.
-   plugins:
-     - jekyll-gist
-     - jekyll-coffeescript
-     - jekyll-assets
-     - another-jekyll-plugin
-   ```
+  * The `_plugins` directory may be changed either directly via the command-line or via the configuration file(s).
+  * Plugins in the `_plugins` directory (or its equivalent(s)) will not be loaded during the `safe` mode.
+  * This route cannot be used to extend the Jekyll CLI.
 
-   Then install your plugins using `gem install jekyll-gist jekyll-coffeescript jekyll-assets another-jekyll-plugin`
+To work with plugins packaged as gems, one has to list the desired gems in the configuration file under a top-level key named `plugins`
+(or `gems` if your Jekyll version is older than `3.5.0`). Additionally, if you're building under `safe` mode, the gem needs to be listed under
+a top-level key named `whitelist`. For example,
 
-3. Add the relevant plugins to a Bundler group in your `Gemfile`. An
-   example:
+```yaml
+plugins:
+  - jekyll-gist
+  - jekyll-coffeescript
+  - jekyll-seo-tag
+  - some-other-jekyll-plugin
 
-   ```ruby
-   group :jekyll_plugins do
-     gem "jekyll-gist"
-     gem "jekyll-coffeescript"
-     gem "jekyll-assets"
-     gem "another-jekyll-plugin"
-   end
-   ```
+# Enable safe mode
+safe: true
 
-   Now you need to install all plugins from your Bundler group by running single command `bundle install`.
+# Whitelist plugins under safe mode.
+# Note that `some-other-jekyll-plugin` is not listed here. Therefore,
+#   it will not be loaded under safe mode.
+whitelist:
+  - jekyll-gist
+  - jekyll-coffeescript
+  - jekyll-seo-tag
+```
+
+In the absence of a Gemfile, one must manually ensure that listed plugins have been installed prior to invoking Jekyll. For example, the
+latest versions of gems in the above list may be installed to a system-wide location by running:
+
+```sh
+gem install jekyll-gist jekyll-coffeescript jekyll-remote-theme some-other-jekyll-plugin
+```
+## Using a Gemfile
+
+The maintenance of various gem dependencies may be greatly simplified by using a Gemfile (usually at the root of site's source) in conjunction
+with a Rubygem named `bundler`. The Gemfile however **should** list all the primary dependencies of your site, including Jekyll itself, not
```

Comment:
"...a Rubygem named Bundler"

Proper nouns FTW.

Reply:
I'm choosing to leave this in lowercase because it references the actual gem-name (`bundler`) instead of the library.
However, the consequent mention has been corrected as requested.

Reply:
> I'm choosing to leave this in lowercase because it references the actual gem-name (`bundler`) instead of the library.

Got it. I was thinking about the distinction between Bundler (the project) v `bundler` (the executable which does a thing). See also Perl and `perl`.

Carry on.